test-fixtures/monorepo/1208-no-double-cloning/spago-one-more-dep.yaml

```diff
@@ -0,0 +1,30 @@
+package:
+  name: deku-consumer
+  dependencies:
+    - deku
+    - deku-core
+    - deku-css
+    - deku-dom
+workspace:
+  packageSet:
+    registry: 56.4.0
+  extraPackages:
+    deku:
+      git: https://github.com/mikesol/purescript-deku.git
+      subdir: deku
+      ref: 276f48adde3d9354f61917f7e9ae2ae7b43df6b2
+    deku-core:
+      git: https://github.com/mikesol/purescript-deku.git
+      subdir: deku-core
+      ref: 98c67533cc8c399aa643b495d3c02bab963e5b80
+    deku-css:
+      git: https://github.com/mikesol/purescript-deku.git
+      subdir: deku-css
+      ref: 4e68a5cec10c91aa3377ce69cc97c276936a1194
+    deku-dom:
+      git: https://github.com/mikesol/purescript-deku.git
+      subdir: deku-dom
+      ref: 6b7c392da7782fe0f2e34811e36b11e630e10b26
```

Comment:
I used Mike's example from #1208, but with specific SHAs. This should be fine until the repo is deleted or its history is rewritten.